source/install/install-kubernetes-mattermost.rst

```diff
@@ -61,7 +78,98 @@ Save the file as ``mattermost-installation.yaml``:
+
+The deployment process can be monitored in the Kubernetes user interface.
+
+**4. Configure DNS and Use Mattermost**
+
+When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get ingress
+
+Copy the resulting hostname or IP address from the ``ADDRESS`` column, open your browser, and connect to Mattermost.
+
+Use your domain registration service to create a canonical name or IP address record for the ``ingressName`` in your manifest,
+pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+
+Navigate to the ``ingressName`` URL in your browser and use Mattermost.
+
+Restoring an Existing Mattermost MySQL Database
+-----------------------------------------------
+
+You can leverage the Mattermost Operator to utilize an existing Mattermost MySQL database with a new Mattermost installation, in its own namespace.
+The steps you follow to create and upload your backup depends on the provider you're using and your use case. It's
+recommended that you consult the relevant documentation or, if your deployment is managed in a different way, consult your Administrator.
+
+It is important to note that this process requires the creation of a new Mattermost
+installation - editing the existing ``.yaml`` files is not recommended and can result in data loss.
+
+The process described below needs to be completed prior to proceeding with the Mattermost deployment.
+
+1. Create a backup of your database (e.g., using *mysqldump*).
+2. Deploy a new server (e.g., an AWS instance).
+3. Install a backup program and back up the database on the new server/instance.
+4. Upload the backed up database to your cloud storage provider (e.g., Amazon S3).
+5. Create a ``secret.yaml``` file:
+
+Open a text editor and create a text file containing your credentials which will be used to access the uploaded database.
+
+Save the file as ``secret.yaml``. The example below is for AWS/S3.
+
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+    name: test-restore
+   type: Opaque
+   stringData:
+    AWS_ACCESS_KEY_ID: XXXXXXXXXXXX
+    AWS_SECRET_ACCESS_KEY: XXXXXXXXXXXX/XXXXXXXXXXXX
+    AWS_REGION: us-east-1
+    S3_PROVIDER: AWS
+
+**Parameters**
+- ``name``. The name of this manifest which is referenced in the installation manifest.
+
+
+6. Create a Mattermost cluster installation manifest.
+
+Open a text editor and create a text file with the following details. Save the file as ``mattermost-installation.yaml``:
+
+.. code-block:: yaml
+
+  apiVersion: mattermost.com/v1alpha1
+  kind: ClusterInstallation
+  metadata:
+    name: mm-example-full
+  spec:
+    size: 5000users
+    ingressName: example.mattermost-example.com
+    ingressAnnotations:
+      kubernetes.io/ingress.class: nginx
+    version: 5.14.0
+    mattermostLicenseSecret: ""
```

Comment:
Should this be the secret created in the initial section detailing installation/configuration? We don't mention this in the steps above so might be a good idea to call it out in the content above?

Reply:
Yes, same secret